cql3/statements/select_statement.cc

```diff
@@ -1660,7 +1660,7 @@ schema_ptr mutation_fragments_select_statement::generate_output_schema(schema_pt
 
 future<exceptions::coordinator_result<service::storage_proxy_coordinator_query_result>>
 mutation_fragments_select_statement::do_query(
-        const locator::node* this_node,
+        locator::host_id this_node,
         service::storage_proxy& sp,
         schema_ptr schema,
         lw_shared_ptr<query::read_command> cmd,
```

Comment:
Not keeping the effective replication map pointer means we can have a tablet or vnode lost under our feet, and return partial responses. It's not critical at all for a debug tool like this, but I'd like to keep a consistent set of guarantees for all statements in terms of data ownership.

Reply:
Good point, will send a v3.